README.md

```diff
@@ -37,6 +37,20 @@ yarn add latex.css
 
 Add any optional classnames to elements with special styles (author subtitle, abstract, lemmas, theorems, etc.). A list of supported class-based elements can be found [here](https://latex.now.sh/#class-based-elements).
 
+## Languages
+
+Theorems, Definitions, Lemmas and Proofs labels can be change to other [supported languages](lang) adding the following code in the head of your project.
```

Comment:
Better: `The labels of theorems, definitions, lemmas and proofs can be changed to other [supported languages](lang) by including the following snippet in addition to the main CSS file.`